harvard-lts/CURIOSity
# Fixing closing <div> for iframe-wrapper.

**Fixing closing `<div>` for iframe-wrapper.**
* * *

**JIRA Ticket**: https://github.com/harvard-lts/CURIOSity/issues/385


# What does this Pull Request do?
This fixes the markup for the iframe-wrapper `</div>`. The closing `<div>` was moved inside an "if" statement for the Mirador Beta Viewer link. In the "No Thumbnail or Viewer"exhibits that If is not "true" so the`<div>` never closes. Because of that all metadata was hidden for items in exhibits that use the No Thumbnail Or Viewer theme because it was in the iframe-wrapper.

# How should this be tested?

A description of what steps someone could take to:
* Build curiosity off of the issue-385 branch
* Harvest one of the exhibits referenced in the issue (Ming, Faculty, Calendar etc)
* Set the exhibit theme to "No Thumbnail Or Viewer"
* Go to any item detail page in the exhibit and confirm that all the metadata shows and the viewer does not show.
* 
# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? No

# Interested parties
@VanessaVenti @mferrarini @dl-maura
